hs-bindgen/src/HsBindgen/C/Tc/Macro.hs

```diff
@@ -296,6 +296,8 @@ data ClassTyCon n where
   IntegralTyCon   :: ClassTyCon ( S Z )
   -- | Class type constructor for Fractional
   FractionalTyCon :: ClassTyCon ( S Z )
```

Comment:
is `Fractional` stuff used for anything anymore?

Reply:
It's used for literals, e.g. `0.123 :: forall a. Fractional a => a`, but probably that can be removed as the C spec says that should just be a double.